.changeset/clean-planets-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
```

Comment:
```suggestion
'@astrojs/cloudflare': patch
```

I am going to flag it as a patch, because it aligns with the behaviour we already mention in the docs